cmd/monaco/generate/deletefile/test-resources/project/notebook-document/config.yaml

```diff
@@ -6,4 +6,4 @@ configs:
     skip: false
   type:
     document:
-      type: notebook-document
+      kind: notebook-document
```

Comment:
why is `kind` not just "notebook" or "dashboard" ?

Reply:
that change is coming in the next PR ;)